### DISCUSSION TITLE
Is it possible to make a custom page in admin using filament?

### DISCUSSION THREAD

**Discussion by faridsa** [filamentphp/filament#6086] [Help] [answered]
Hello fellas, I recently discovered filament and it really captivated me. So I decided to use it as soon as possible. The opportunity is about refactoring an admin panel for an online catalog of products and materials originally made using php5.
Currently I have achieved almost everything to perfection. But, there is always a but, there is a particular list of materials that the client wants to keep as is in its look and feel and I am not finding a way to solve it with my actual knowledge of this framework.

The model is a table of a material type that relates colors and thicknesses with prices, as seen in the attached image.
But also each line must have an editable price field to be able to perform an inline update.
Additionally we have a percentage field that should allow a massive modification of all the prices listed in the table.

![relationship(1)](https://user-images.githubusercontent.com/23212965/229193018-331d7bac-66e0-426f-91e9-e59759396fdb.png)

I can quickly achieve this goal using laravel blade, but I would like to solve it with filament or eventually by embedding a custom page inside a link accessible from filament so as not to lose the rest of the existing features in the framework. Is that possible?

Thank you in advance for any suggestion!

**Comment by danharrin** [comment, marked as answer]
You can use the TextInputColumn on tables for the individual changes. You can use a bulk action for mass-updates.

**Reply by faridsa** [reply]
Thank you Dan, you make my day. I was looking deeper on the excelente filament docs and found the way based in your suggestions.
![Screenshot from 2023-04-01 20-11-50](https://user-images.githubusercontent.com/23212965/229320168-54590a8a-7ef9-4be7-8478-e7e2042d5a67.png)
